XamlControlsGallery/DataModel/ControlInfoData.json

```
@@ -1957,8 +1957,8 @@
               "Uri": "https://www.docs.microsoft.com/uwp/api/microsoft.ui.xaml.controls.animatedicon"
             },
             {
-              "Title": "AnimatedIcon - Guidance ",
-              "Uri": "https://docs.microsoft.com/windows/uwp/design/style/animated-icon"
+              "Title": "AnimatedIcon - Guidance",
+              "Uri": "https://docs.microsoft.com/en-us/windows/apps/design/controls/animated-icon"
```

```suggestion
              "Uri": "https://docs.microsoft.com/windows/apps/design/controls/animated-icon"
```

We should not include locales in the URL as this can prevent the page being opened in the language native for the user.